NGitLab/Models/Variable.cs

```diff
@@ -19,6 +27,16 @@ public class Variable
     [JsonPropertyName("masked")]
     public bool Masked { get; set; }
 
+    [JsonPropertyName("raw")]
+    public bool Raw { get; set; }
+
+    /// <summary>
+    /// The environment scope of a variable
+    /// </summary>
+    /// <remarks>
+    /// Create and Update of project variable: All tiers (Free, Premium, Ultimate).<br/>
+    /// Create and Update of group variable: Premium and Ultimate only.
+    /// </remarks>
     [JsonPropertyName("environment_scope")]
```

Comment:
I may have missed this one, renaming = breaking change, though, as EnvironmentScope makes more sense on both this class and the others (Create/Update), what we can do is:

```csharp
[EditorBrowsable(EditorBrowsableState.Never)]
public string Scope
{
    get => EnvironmentScope;
    set => EnvironmentScope = value;
}

/// <summary>
/// The environment scope of a variable
/// </summary>
/// <remarks>
/// Create and Update of project variable: All tiers (Free, Premium, Ultimate).<br/>
/// Create and Update of group variable: Premium and Ultimate only.
/// </remarks>
[JsonPropertyName("environment_scope")]
public string EnvironmentScope { get; set; }
```

Reply:
Addressed by [143c88e](https://github.com/ubisoft/NGitLab/pull/697/commits/143c88eccdffc05dc0a5828776c0f572c460fd3c)